flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/runtime/utils/UserDefinedFunctionTestUtils.scala

```diff
@@ -377,6 +377,13 @@ object UserDefinedFunctionTestUtils {
     override def getResultType(signature: Array[Class[_]]): TypeInformation[_] = Types.JAVA_BIG_DEC
   }
 
+  @SerialVersionUID(1L)
+  object MyIsNull extends ScalarFunction {
```

Comment:
maybe we could improve this name: `MyIsNull` ?  
like `IsNullFunction` or `IsNullUDF` or `IsNullScalarFunction` ?

Reply:
Hmm, `IsNullUDF ` sounds better to me